files/galaxy/tpv/destinations.yml

```diff
@@ -48,35 +50,33 @@ destinations:
 
   interactive_pulsar_gpu:
     inherits: interactive_pulsar
-    max_accepted_gpus: 1
+    min_accepted_gpus: 1
+    max_accepted_gpus: 9999
```

Comment:
`max_accepted_gpus` is set to 9999 instead of null or left empty because of how inheritance works (values for less general entities cannot be set to none).